interaction.go

```diff
@@ -47,6 +48,13 @@ const (
 	InteractionCallbackUpdateMessage
 )
 
+type Interactable interface {
```

Comment:
Do we need this? Can't we just specify the exact parameter type instead?

Reply:
`Session.SendInteraction()` can't be a generic function in the interface, so an interface needs to be passed, unless you prefer multiple iterations of `Session.SendInteraction()` for every type parameter of `InteractionData[T]`.

Reply:
ah fair